traitcuration/settings.py

```diff
@@ -9,7 +9,7 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/3.0/ref/settings/
 """
-
+import django_heroku
```

Comment:
```suggestion

import django_heroku
```

Adding a blank line between the end of docstring and the first statement